src/common/file_util.h

```diff
@@ -46,6 +46,13 @@ struct FSTEntry {
     std::vector<FSTEntry> children;
 };
 
+// Timestamps for GetFileTimeStampRaw
+struct FileTimeStampRaw {
```

Comment:
```suggestion
struct Timestamp {
```
imo File is redundant as its in the FileUtil namespace
also Raw doesn't really convey any addt'l info either